files/bridgesi/sprint-1-1_18inchPipePile_bridgeSI.ifc

```diff
@@ -0,0 +1,101 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION ((''), '2;1');
+FILE_NAME ('', '2020-09-17T00:38:34', (''), (''), '', '', '');
+FILE_SCHEMA (('IFC4'));
```

Comment:
Remember to update the schema reference when the checker actually checks this. Not needed just now.

Reply:
Much appreciated, have resolved this by editing line 5.